src/almanack/cli.py

```diff
@@ -3,20 +3,155 @@
 """
 
 import json
+import shutil
+import sys
+from datetime import datetime, timezone
 
 import fire
+from tabulate import tabulate
 
-from .metrics.data import get_table
+from .metrics.data import (
+    _get_almanack_version,
+    gather_failed_almanack_metrics,
+    get_table,
+)
 
 
-def cli_get_table() -> None:
+class AlmanackCLI(object):
     """
-    Creates Fire CLI for get_table_json_wrapper.
+    Almanack CLI class for Google Fire
```

Comment:
I like this class! It might be helpful to add a brief note on functionality here in the docstring. For example, it would be nice to outline what the difference is between `table` and `check`

Reply:
Thanks, good idea, I'll add that in!

Reply:
Added a description to the class docstring.